src/Ocelot.Provider.Polly/OcelotBuilderExtensions.cs

```diff
@@ -1,18 +1,29 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
+
 using Ocelot.Configuration;
 using Ocelot.DependencyInjection;
 using Ocelot.Errors;
 using Ocelot.Logging;
 using Ocelot.Provider.Polly.Interfaces;
 using Ocelot.Requester;
+
 using Polly.CircuitBreaker;
 using Polly.Timeout;
 
 namespace Ocelot.Provider.Polly;
 
 public static class OcelotBuilderExtensions
 {
+    public static IOcelotBuilder AddPolly<T>(this IOcelotBuilder builder, QosDelegatingHandlerDelegate delegatingHandler, IDictionary<Type, Func<Exception, Error>> errorMapping) where T : class, IPollyQoSProvider<HttpResponseMessage>
```

Comment:
Too long line of code! Formatting issue.
It is better to put each parameter on new line.
`where` on new line too.